jrportfolio.css

```diff
@@ -301,55 +305,62 @@ html {
 		margin-top: -1rem;
 		padding: 10px;
 	}
-/* MQ's for portfolio section */
+/* MQ's for large image section */
 	.bigpicturecontainer .content {
 	    padding-top: 211px;
 	    padding-bottom: 183px;
 	}
 
 	.bigpicturecontainer h3 {
 	    line-height: 1.1rem;
-}
+	}
+	/* MQ's for about me and contact me sections */
+	.myinfocontainer {
+		font-family: "trebuchet ms", sans-serif;
+		width: 100%;
+		margin: 0 auto;
+		overflow: auto;
+	}
 
-.box-3container {
-	width: 100%;
-}
+	.buttoncontainer {
+		width: 100%;
+		margin-top: 34px;
+		text-align: center;
+	}
 
-.box-1container {
-	width: 100%;
-	height: auto;
-}
+	.personalinfo {
+		width: 85%;
+		font-size: 1.6rem;
+		margin-top: 17px;
+	}
+
+	.personalinfo p {
+		line-height: 36px;
+	}
+	/*MQ's for portfolio section*/
+	.box-3container {
+		width: 100%;
+	}
+
+	.box-1container {
+		padding: 0 20px;
+	}
 }
 /*Media queries for up to 768px*/
 @media screen and (max-width:768px) {
 	.show-for-small{display:block;}
 	.show-for-large{display:none;}
 /* MQ's for portfolio section */
+	.box-1container {
+		width: 100%;
+	}
 	.bigpicturecontainer .content {
 	    padding-top: 122px;
 	    padding-bottom: 66px;
 	}
 /* MQ's for about me and contact me sections */
-.myinfocontainer {
-	width: 100%;
-	margin: 0 auto;
-	overflow: auto;
-}
-
-.buttoncontainer {
+	.buttoncontainer {
```

Comment:
Check this and make sure it's indented correctly.

Reply:
Done